packages/plc/src/server/db.ts

```diff
@@ -1,44 +1,59 @@
+import { Kysely, SqliteDialect } from 'kysely'
+import SqliteDB from 'better-sqlite3'
 import { CID } from 'multiformats/cid'
 import { cidForData } from '@adxp/common'
-import {
-  EntityManager,
-  DataSource,
-  Entity,
-  Column,
-  PrimaryColumn,
-  In,
-  Not,
-} from 'typeorm'
-import { Mutex } from 'async-mutex'
 import * as document from '../lib/document'
 import * as t from '../lib/types'
 import { ServerError } from './error'
 
-export class Database {
-  db: SafeDB
+interface OperationsTable {
+  did: string
+  operation: string
+  cid: string
+  nullified: 0 | 1
```

Comment:
better-sqlite3 forces you to be honest about sqlite values & doesn't let you use booleans 😛